src/platforms/android/configuration/app-not-respond.mdx

```diff
@@ -47,32 +49,81 @@ You can also specify how long the thread should be blocked before the ANR is rep
 </application>
 ```
 
-### ApplicationExitInfo (v2)
+## ApplicationExitInfo (v2)
 
 This approach reads the [ApplicationExitInfo](https://developer.android.com/reference/android/app/ApplicationExitInfo) API on the next app launch
 and asynchronously sends ANR events to Sentry for each ANR in the history, enriching only the latest one with breadcrumbs, contexts, tags, etc.
 
 The integration reports ANR events with `mechanism:AppExitInfo`.
 
+<Note>
+
+In case [ApplicationExitInfo#getTraceInputStream](<https://developer.android.com/reference/android/app/ApplicationExitInfo#getTraceInputStream()>) returns `null`, the SDK no longer reports an ANR event as these events are not actionable without it.
+
```

Comment:
```suggestion
If [ApplicationExitInfo#getTraceInputStream](<https://developer.android.com/reference/android/app/ApplicationExitInfo#getTraceInputStream()>) returns `null`, the SDK will no longer report an ANR event, since these events won't be actionable without it.

```